src/AppBundle/Controller/Pa/TeamController.php

```diff
@@ -62,6 +63,10 @@ public function editAction(Request $request, $id)
     {
         $user = $this->getRestClient()->get('team/member/'.$id, 'User');
 
+        if (!$this->getUser()->canEditUser($user)) {
```

Comment:
that could be a security.yml rule. The API also should be implemented not to give access to edit users depending on the roles